packages/composites/por-indexer/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { makeExecute } from './adapter'
 import { expose } from '@chainlink/ea-bootstrap'
+import { NAME } from './config'
 
-const NAME = 'POR_INDEXER'
-export = { NAME, makeExecute, ...expose(NAME, makeExecute()) }
+const adapterContext = { name: NAME }
+export = { NAME, makeExecute, ...expose(adapterContext, makeExecute()) }
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute())
export { NAME, makeConfig, makeExecute, server }
```